tools/main.go

```diff
@@ -110,33 +119,39 @@ func convertConfigImageToRegsyncImages(repo config.Repository, image *config.Ima
 }
 
 func migrateImagesList(_ context.Context, cmd *cli.Command) error {
-	imagesListPath := cmd.Args().Get(0)
-	if imagesListPath == "" {
-		return fmt.Errorf("must pass path to images list file as argument")
+	if cmd.Args().Len() != 2 {
+		return fmt.Errorf("must pass source and target image")
 	}
+	sourceImage := cmd.Args().Get(0)
+	targetImage := cmd.Args().Get(1)
 
 	configYaml, err := config.Parse(configYamlPath)
 	if err != nil {
-		return fmt.Errorf("failed to parse %q: %w", configYamlPath, err)
+		return fmt.Errorf("failed to parse config: %w", err)
 	}
 	accumulator := config.NewImageAccumulator()
 	for _, existingImage := range configYaml.Images {
 		accumulator.AddImage(existingImage)
 	}
 
-	legacyImages, err := legacy.ParseImagesList(imagesListPath)
+	imagesListComment, legacyImages, err := legacy.ParseImagesList(imagesListPath)
 	if err != nil {
-		return fmt.Errorf("failed to parse %q: %w", imagesListPath, err)
+		return fmt.Errorf("failed to parse images list: %w", err)
 	}
 
 	configJson, err := legacy.ParseConfig(configJsonPath)
 	if err != nil {
 		return fmt.Errorf("failed to parse %q: %w", configJsonPath, err)
 	}
 
+	if configJson.Contains(sourceImage) {
+		log.Printf("warning: %s refers to image with source %q", configJsonPath, sourceImage)
```

Comment:
`log.Printf("warning: %s refers to image with source %q \n", configJsonPath, sourceImage)`

Reply:
The newline isn't necessary when using `log.Printf()` - the logging package ensures every log statement is on a separate line. See https://go.dev/play/p/T8tbtUuB7Tb

Reply:
Actually, I'm simply going to stop using the `log` package.